### PR TITLE
Add cras specific playback pipeline and topology

### PR DESCRIPTION
**Pull request by ranj063**

### CODE REVIEW

**Comment by ranj063** [comment]
@lgirdwood @cujomalainey @keyonjie I've updated the PR to create a separate pipeline with cras capabilities req and use it.

**Comment by cujomalainey** [comment]
@dgreid FYI

File: topology/sof-yorp.m4 [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+# capture DAI is SSP1 using 2 periods
+# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-capture.m4,
+	3, SSP, 2, SSP2-Codec,
+	PIPELINE_SINK_3, 2, s16le,
+	48, 1000, 0, 0)
+
+# capture DAI is DMIC0 using 2 periods
+# Buffers use s32le format, with 48 frame per 1000us on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-capture.m4,
```
Comment (cujomalainey) [inline topology/sof-yorp.m4]: Can we get capture streams as well that use 48k?

Comment (ranj063) [reply topology/sof-yorp.m4]: @cujomalainey the current dmic capture sample rate is 48K. Are you asking for it to be changed?

**Comment by keqiaozhang** [comment]
@ranj063 
This PR will cause IPC timeout. Please check.

**Comment by ranj063** [comment]
@keqiaozhang thanks for checking. Yes, it does seem to cause errors while starting audio the second time around. I will close this PR now and re-open when we have resolution.

**Pull request closed without merging**